### PR TITLE
Upload script

### PR DESCRIPTION
**Pull request by mrchucho**
@natemcdowel I wasn't sure where the best place for this script would be, but this makes a decent start. The "script" is actually a Ruby gem, so we could certainly split it out to its own repo, too.

Anyway, this is the first part of [PUI-6135](https://vitals.atlassian.net/browse/PUI-6135). Given a PDF, this will upload it to a client-specific S3 bucket. That bucket is in a subset of the existing PDF directory, so it's already "serve-able".

```
./bin/upload.rb <client> <pdf>
# uploads to https://<env>.<client>.<domain>/pdf_requests/<pdf>
```

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+source 'https://rubygems.org'
+
+gemspec
+gem 'platform-configuration', git: 'git@github.com:mdx-dev/platform-configuration.git', branch: 'master'
```
Comment (tobowers) [inline Gemfile]: generally I like to put this dependency in the gemspec as well. The *user* of the repo still needs to make sure they include both gems, but at least it errors when the dependency isn't met.

Comment (mrchucho) [inline Gemfile]: Good suggestion! I remember this was discussed recently, but I'd forgotten where we wound up.

**Comment by mrchucho** [comment]
@natemcdowel any input?

**Comment by mrchucho** [comment]
@natemcdowel should be good to merge this too.

**Comment by natemcdowel** [comment]
:+1: This looks good to me @mrchucho .  As we discussed, we'll just need to add a last task to kick off the UI PDF app.

**Pull request merged**